app/helpers/application_helper.rb

```diff
@@ -19,4 +19,8 @@ def a11y_action_link(text, href, context = "")
       link_to("#{text} #{raw(span)}".html_safe, href, class: "govuk-link")
     end
   end
+
+  def link_to_new_window(text, href)
+    link_to("#{text} (opens in new window)", href, class: "govuk-link", target: "_blank")
```

Comment:
I think we'd probably be better saying "opens in new tab" here, as the default behaviour for `_blank` is to open a new tab nowadays. Also [GOV.UK Design system](https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab) says to: 

> "Include `rel="noreferrer noopener"` along with `target="_blank"` to reduce the risk of [reverse tabnabbing](https://owasp.org/www-community/attacks/Reverse_Tabnabbing)."

Reply:
Thank you for catching this @pezholio !